### PR TITLE
[CI] Moved to Github Actions

### PR DESCRIPTION
**Pull request by mnocon**
Moved CI to Github Actions

In order to run tests on PHP 8 I had to upgrade php-cs-fixer - I've decided that it's a good time to migrate from the current CS configuration to the shared one.

### CODE REVIEW

**Review by tomaszszopinski — APPROVED** [review]

**Review by micszo — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    pull_request: ~
+
+jobs:
+    cs-fix:
+        name: Run code style check
+        runs-on: "ubuntu-20.04"
+        strategy:
+            matrix:
+                php:
+                    - '7.4'
```
Comment (konradoboza) [inline .github/workflows/ci.yaml]:
```suggestion
                    - '8.0'
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        name: Unit tests
+        runs-on: "ubuntu-20.04"
+        timeout-minutes: 15
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php:
+                    - '7.3'
+                    - '7.4'
```
Comment (konradoboza) [inline .github/workflows/ci.yaml:46]:
```suggestion
                    - '7.4'
                    - '8.0'
                    - '8.1'
```

**Review by konradoboza — CHANGES_REQUESTED** [review]

**Review by konradoboza — APPROVED** [review]

File: composer.json [hunk 3]
```diff
@@ -53,5 +56,10 @@
     },
     "conflict": {
         "lolautruche/ez-core-extra-bundle": "<2.0"
+    },
+    "config": {
+        "allow-plugins": {
+            "composer/package-versions-deprecated": true
```
Comment (alongosz) [inline composer.json:62]: probably required for all LTS-supported versions

**Review by alongosz — APPROVED** [review]

File: composer.json (hunk 3 above)
Comment (mnocon) [reply composer.json:62]: Yep, I will be going through all v2 repos when migrating them to GA - will be added then

**Pull request merged**